### ISSUE TITLE
Compression Limited Barrel

### ISSUE THREAD

**Issue by Suragi** [P3pp3rF1y/SophisticatedStorage#147]
When connecting a 3 limited barrell to a wall with a controller the controller cannot interact with it very well. Nothing will go into the drawing through a controller. Can use normally otherwise.

**Comment by P3pp3rF1y** [comment]
I will need more info on steps how to recreate this. Just tried testing and I am able to put stuff both manually and with hopper through controller into limited barrel with compression upgrade. So will need exact steps how I can recreate the issue by putting blocks together from scratch.

**Comment by Suragi** [comment]
I will see what i can do about getting you more detailed info. I will also include my modlist with the update. It could be a weird conflict.

**Comment by Suragi** [comment]
So after some testing. Loading up with just the mod itself and JEI. The compression upgrade works until world reboot. Then it has to be manually interacted with first before it will work with the controller (double right click). I should clarify all tests done with the controller were done using double right click or right click with the item in hand. No automation used.

On my pack I was able to gain success for some reason by turning off the following mods.

Immersive Portals
Dungeons Gear
Dungeons Libraries
Storage Labels

I do not know why turning off these 3 mods returned the compression drawers to the same state as mentioned above. But it does. I have attached a Curseforge profile containing all the mods and scripts of the pack.

Minecraft Version: 1.19.2
Forge Version: 43.2.7
Mod Version: 23.307
Library Version: 3.18.43.801

[SOGECraft Test 1.19.2.zip](https://github.com/P3pp3rF1y/SophisticatedStorage/files/11245337/SOGECraft.Test.1.19.2.zip)
